tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+import os
+import pytest
+import requests
+import shutil
+from zipfile import ZipFile
+from . import fixture_path
+
+
+def download(url: str, to_path: str):
+    r = requests.get(url, stream=True)
+    os.makedirs(os.path.dirname(to_path), exist_ok=True)
+    with open(to_path, "wb") as f:
+        for chunk in r.iter_content(chunk_size=1024):
+            if chunk:
+                f.write(chunk)
+
+
+@pytest.fixture(autouse=True)
+def word2vec() -> str:
```

Comment:
since a specific word2vec file is required/paired to a version of the code, we should extract out `word2vec_download()` as a global module function, and then use that function both to download here *and* for the similar fixture in `opentutor-classifier-api`

The function could be like this

```python
# this function would be part of the module itself,
# so that anything that pip installs opentutor-classifier
# can also download the word2vec this way
def word2vec_download(to_path='shared', replace_existing = True) -> string:
    word2vec_path = os.path.join(to_path, "word2vec.bin")
    if os.path.isfile(word2vec_path):
        return word2vec_path
    word2vec_zip = os.path.join(shared, "word2vec.zip")
    download("http://vectors.nlpl.eu/repository/20/6.zip", word2vec_zip)
    with ZipFile(word2vec_zip, "r") as z:
        z.extract("model.bin")
    shutil.move("model.bin", word2vec_path)
    os.remove(word2vec_zip)
    return word2vec_path
```

...and then the fixture (here and in `opentutor-classifier-api` could just be

```python
@pytest.fixture(autouse=True)
def word2vec() -> str:
    return word2vec_download(
        to_path=fixture_path(os.path.join("shared", "word2vec.bin")), 
        replace_existing=False
    )